openupgrade_scripts/scripts/website_crm/15.0.2.1/post-migration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def migrate_website_crm_views(env):
+    """Migrate website_crm views in the post-migration process so that the new contact form
+    views have been defined.
+    This function checks if the "website_crm" module is installed. If it is, it looks for
+    the views associated with the website and updates the arch_db field replacing the
+    "mail.mail" occurrences with "crm.lead". This is done to adapt the
```

Comment:
The only view changing is the contact form, so there's no need to generalize
```suggestion
    """This is done to adapt the
```

Reply:
thanks